src/main/java/com/chat/yourway/controller/ContactController.java

```diff
@@ -0,0 +1,31 @@
+package com.chat.yourway.controller;
+
+import com.chat.yourway.dto.request.ChangePasswordDto;
+import com.chat.yourway.service.ContactService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Tag(name = "Contact")
+@RestController
+@RequestMapping("/contacts")
+@RequiredArgsConstructor
+public class ContactController {
+
+    private final ContactService contactService;
+
+    @Operation(summary = "Change to new password")
+    @PatchMapping(path = "/password",
+            produces = APPLICATION_JSON_VALUE,
```

Comment:
as i can see this method returns void - so it means that there is no sense to use produces = APPLICATION_JSON_VALUE